google-cloud-spanner/docs/toc.json

```diff
@@ -31,7 +31,25 @@
   "services": [
     {
       "title": "Spanner",
-      "type": "google/cloud/spanner"
+      "type": "google/cloud/spanner",
+      "patterns": ["\/spanner", "google\/protobuf"],
+      "nav": [
+        {
+          "title": "V1",
+          "type": "google/cloud/spanner/v1",
+          "patterns": ["\/spanner\/v1", "google\/protobuf"],
+          "nav": [
+            {
+              "title": "SpannerClient",
```

Comment:
Admin/Instances/V1/InstanceClient and Admin/Databases/V1/DatabaseClient should also be listed here.